code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -390,8 +390,9 @@
 
 	if(print_flavor_text()) msg += "[print_flavor_text()]\n"
 
-	if (GLOB.config.allow_Metadata && client.prefs.metadata)
-		msg += "<span class='deadsay'>OOC Notes:</span> <a href='?src=\ref[src];metadata=1'>\[View\]</a>\n"
+	if(!isnull(client))
+		if(GLOB.config.allow_Metadata && client.prefs.metadata)
```

Comment:
You can simplify this with the conditional access operator:

```suggestion
	if(GLOB.config.allow_Metadata && client?.prefs.metadata)
```

Reply:
Oh, I totally forgot that DreamMaker has them. Thank you!